src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -127,7 +127,7 @@ def default_api_version(self):
         ResourceType.MGMT_NETWORK: '2019-11-01',
         ResourceType.MGMT_COMPUTE: SDKProfile('2019-07-01', {
             'resource_skus': '2019-04-01',
-            'disks': '2019-07-01',
+            'disks': '2019-11-01',
```

Comment:
have we done full regression :)

Reply:
underway

Reply:
Done. Can't guarantee no error in live test. But no new error in disk related tests.